### PR TITLE
[Circletracer] Add html file for circle file visualization

### PR DESCRIPTION
**Pull request by yesfordev**
This is the default index.html file for circle file visualization.

ONE-vscode-DCO-1.0-Signed-off-by: Youngeun Seo <yesfordev@gmail.com>

### CODE REVIEW

**Comment by yesfordev** [comment]
from draft #179
from issue #66.

File: media/Circletracer/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <!--
+					Use a content security policy to only allow loading images from https or from our extension directory,
+					and only allow scripts that have a specific nonce.
+				-->
+    <meta http-equiv="Content-Security-Policy"
+        content="default-src 'none'; style-src _webview.cspSource; img-src _webview.cspSource https: http: data: blob:; script-src 'unsafe-inline' data: blob: http: https:; connect-src _webview.cspSource https:">
```
Comment (CODe1995) [inline media/Circletracer/index.html:11]: I think we should use our `nonce` to increase security.
Although the `nonce` value is used now, it is not being reviewed in the security policy.

Comment (llFreetimell) [reply media/Circletracer/index.html:11]: Is there any case that we need `http`, `blob` or other things except accessing to `https://d3js.org/d3.v5.min.js`?

**Comment by llFreetimell** [comment]
Is this file is related with visualizing circle graph? 
I am asking because of copyright statement :)

**Comment by yesfordev** [comment]
> Is this file is related with visualizing circle graph? I am asking because of copyright statement :)

Yes. Copyright must be filled in. :) How should I write it?

**Comment by llFreetimell** [comment]
@yesfordev 

You can refer to https://github.com/Samsung/ONE-vscode/pull/240#discussion_r721870874 for the content.
And you can check detail at /media/Jsontracer/index.html` :D

**Comment by yesfordev** [comment]
I add a license and change code like
`openFileSelector('1') -> openFileSelector(1)`

File: media/Circletracer/index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+// MIT License
```
Comment (llFreetimell) [inline media/Circletracer/index.html:4]: `html` also accept these comment format?

Comment (yesfordev) [reply media/Circletracer/index.html:4]: If there is a problem with the html format, I will change it to the <!-- --!>` format.

**Comment by llFreetimell** [comment]
Please check https://github.com/Samsung/ONE-vscode/pull/237#discussion_r721865598

File: media/Circletracer/index.html (hunk 1 above)
Comment (yesfordev) [reply media/Circletracer/index.html]: I change security policy setting. :)

File: media/Circletracer/index.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
```
Comment (llFreetimell) [inline media/Circletracer/index.html:25]:
```suggestion
-->

<!--
```

File: media/Circletracer/index.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+</head>
+
+<body>
+    <div id="root">
+        <main>
+            <h1 id="title">Circle Tracer</h1>
+            <div id="main">
+                <nav>
+                    <div class="left-btns">
+                        <button id="first-json-btn" class="json-load-btn" onclick="openFileSelector(1);">Load First
```
Comment (llFreetimell) [inline media/Circletracer/index.html:58]: We need something like...

```suggestion
                        <button id="first-json-btn" class="json-load-btn" onclick="openFileSelector("#first-json-btn");">Load First
```

according to https://github.com/Samsung/ONE-vscode/pull/236#discussion_r721860995

File: media/Circletracer/index.html [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+                            Json File</button>
+                        <button id="second-json-btn" class="json-load-btn" onclick="openFileSelector(2);">Load Second
```
Comment (llFreetimell) [inline media/Circletracer/index.html:60]: ditto

**Review by llFreetimell — APPROVED** [review]
LGTM

**Review by donggillee-dev — APPROVED** [review]
LGTM

**Comment by llFreetimell** [comment]
As @CODe1995 is out of office, merge without his approval

**Pull request merged**